### DISCUSSION TITLE
Got an ImportError while importing atoti.web

### DISCUSSION THREAD

**Discussion by QihaMarz** [atoti/atoti#776] [Q&A]
Hello,

I got an error while importing the `atoti.web` although I've downloaded the latest version of atoti which is 0.7.4. Anyone encountered the same thing? How to solve it?

**Environment**:
Python - 3.9.12
Win64

**Comment by tibdex** [comment]
Hi Faqiha, there's no such thing as `atoti.web`. Do you have some code to share so that we can reproduce your issue?
